receiver/awscontainerinsightreceiver/README.md

```diff
@@ -429,6 +428,21 @@ kubectl apply -f config.yaml
 
 <br/><br/> 
 
+### Cluster Resources
+| Metric                                  | Unit    |
+|-----------------------------------------|---------|
+| apiserver_storage_list_duration_seconds | Seconds |
```

Comment:
should this be part of existing `Cluster` metrics?

Reply:
That's where I put it initially but this metric is also emitted at a resource level so its in it own table (similar to etcd_db_total_size_in_bytes)